a10_octavia/controller/worker/controller_worker.py

```diff
@@ -312,10 +354,42 @@ def create_load_balancer(self, load_balancer_id):
         store[constants.UPDATE_DICT] = {
             constants.LOADBALANCER_TOPOLOGY: topology
         }
+        validation_flag=False
+        flag=False
+        if self.config.has_option("RACK_VTHUNDER","devices") and self.config.has_section("RACK_VTHUNDER"):
+            project_conf = self.config.get('RACK_VTHUNDER', 'devices')
+            rack_conf = eval(project_conf.strip('"'))
+            for i in range(len((rack_conf["device_list"]))):
+                project_id = rack_conf["device_list"][i]["project_id"]
+                if  project_id == lb.project_id:
+                    flag=True
+                    ip_address = rack_conf["device_list"][i]["ip_address"]
+                    undercloud = bool(rack_conf["device_list"][i]["undercloud"])
+                    username = rack_conf["device_list"][i]["username"]
+                    password = rack_conf["device_list"][i]["password"]
+                    device_name = rack_conf["device_list"][i]["device_name"]
+                    axapi_version = rack_conf["device_list"][i]["axapi_version"]
+                    role = rack_conf["device_list"][i]["role"]
+                    topology = rack_conf["device_list"][i]["topology"]
+                    vthunder_conf = data_models.VThunder(project_id=project_id, ip_address=ip_address, undercloud=undercloud, 
+                                                         username=username, role=role, topology=topology,
+                                                         password=password,device_name=device_name, axapi_version=axapi_version)
+                    validation_flag = self.validate(vthunder_conf)
+                    break
+
+        if flag:
+            if validation_flag:
+                LOG.info("A10ControllerWorker.create_load_balancer fetched project_id : %s from config file for Rack Vthunder" % (project_id))
+                create_lb_flow = self._lb_flows.get_create_rack_vthunder_load_balancer_flow(
+                    vthunder_conf=vthunder_conf, topology=topology, listeners=lb.listeners)
+                create_lb_tf = self._taskflow_load(create_lb_flow, store=store)
+            else:
+                LOG.info("Validations Failed")
+        else:
+            create_lb_flow = self._lb_flows.get_create_load_balancer_flow(
```

Comment:
In A10 config we should create a hashmap of the project ids with racked devices. This way we can just look it up in the hashmap. And also, not have to load the device config every time.

We shouldn't be cycling over the entire racked device list each time we go to create an amphora on a tenant that isn't actually using a racked device.